documentation/docs/05-load.md

```diff
@@ -2,253 +2,282 @@
+	</ul>
+</aside>
 ```
 
-In other words `+page.server.js` passes `data` along to `+page.js`, which passes `data` along to `+page.svelte`.
+Data returned from layout `load` functions is available to child `+layout.svelte` components and the `+page.svelte` component as well as the layout that it 'belongs' to.
 
-#### params
+```diff
+/// file: src/routes/blog/[slug]/+page.svelte
+<script>
++	import { page } from '$app/stores';
 
-`params` is derived from `url.pathname` and the route filename.
+	/** @type {import('./$types').PageData} */
+	export let data;
 
-For a route filename example like `src/routes/a/[b]/[...c]` and a `url.pathname` of `/a/x/y/z`, the `params` object would look like this:
++	// we can access `data.posts` because it's returned from
++	// the parent layout `load` function
++	$: index = data.posts.findIndex(post => post.slug === $page.params.slug);
++	$: next = data.posts[index - 1];
+</script>
 
-```json
-{
-	"b": "x",
-	"c": "y/z"
-}
+<h1>{data.post.title}</h1>
+<div>{@html data.post.content}</div>
+
++{#if next}
++	<p>Next post: <a href="/blog/{next.slug}">{next.title}</a></p>
++{/if}
 ```
 
-#### routeId
+> If multiple `load` functions return data with the same key, the last one 'wins'.
 
-The name of the current route directory, relative to `src/routes`:
+### $page.data
 
-```js
-/// file: src/routes/blog/[slug]/+page.js
-/** @type {import('./$types').PageLoad} */
-export function load({ routeId }) {
-	console.log(routeId); // 'blog/[slug]'
-}
-```
+The `+page.svelte` component, and each `+layout.svelte` component above it, has access to its own data plus all the data from its parents.
 
-#### url
+In some cases, we might need the opposite — a parent layout might need to access page data or data from a child layout. For example, the root layout might want to access a `title` property returned from a `load` function in `+page.js` or `+page.server.js`. This can be done with `$page.data`:
 
-An instance of [`URL`](https://developer.mozilla.org/en-US/docs/Web/API/URL), containing properties like the `origin`, `hostname`, `pathname` and `searchParams` (which contains the parsed query string as a [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) object). `url.hash` cannot be accessed during `load`, since it is unavailable on the server.
+```svelte
+/// file: src/routes/+layout.svelte
+<script>
+	import { page } from '$app/stores';
+</script>
 
-> In some environments this is derived from request headers during server-side rendering. If you're using [adapter-node](/docs/adapters#supported-environments-node-js), for example, you may need to configure the adapter in order for the URL to be correct.
+<svelte:head>
+	<title>{$page.data.title}</title>
+</svelte:head>
+```
 
-### Input methods
+Type information for `$page.data` is provided by `App.PageData`.
 
-`LoadEvent` also has the following methods:
+### Shared vs server
 
-#### depends
+As we've seen, there are two types of `load` function:
```

Comment:
```suggestion
As we've seen, there are two types of `load` functions:
```

Reply:
`function` singular is correct!

Reply:
I've seen PRs before for the svelte.dev docs that wanted to make this sort of grammar change. I think the singular might just be intentional and be Rich's style. It also feels a little bit more formal/British to me.

Reply:
Whoops, thanks for not letting me know Rich had already commented, GitHub.